test/fixtures/definitions.txt

```diff
@@ -83,5 +153,176 @@ text with **inlines**, \[backslash escapes\], typography(tm), "smart quotes", ch
     text with **inlines**, \[backslash escapes\], typography(tm), "smart quotes", character references (&#x1f600;) &amp;&nbsp;entities.
 .
 <p>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</p>
-<blockquote>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</blockquote>
+<blockquote>
+<p>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</p>
+</blockquote>
+.
+
+Thematic breaks and headings should be rendered within indented quotes
+.
+    * * *
+
+    ---
+
+    # ATX Heading
+
+    SEText Heading
+    --------------
+.
+<blockquote><hr /></blockquote>
+<blockquote><hr /></blockquote>
+<blockquote><h1>ATX Heading</h1></blockquote>
+<blockquote><h2>SEText Heading</h2></blockquote>
+.
+
+Indented quotes separated by blank indented lines (spaces or tabs) are part of the same block quote.
+.
+    chunk1
+    
+      chunk2
+.
+<blockquote>
+<p>chunk1</p>
+<p>chunk2</p>
+</blockquote>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when not separated from the previous paragraph.
```

Comment:
HTML does not allow block quotes within a paragraph, but it happens often within literature. This was my attempt to address the occurrence. Thoughts?